docusaurus/docs/cloud/advanced/database.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Database
+displayed_sidebar: cloudSidebar
+description: Configure your own database on Strapi Cloud.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/database.html
+---
+
+# Database
+
+Strapi Cloud provides a pre-configured PostgreSQL database by default. However, you can also configure it to utilize an external SQL database, if needed.
+
+:::caution
+While supported, it is not recommended to use an external database with Strapi Cloud, unless there is an explicit need to do so. This is because Strapi Cloud provides a managed database that is optimized for Strapi. Using an external database may result in unexpected behavior and/or performance issues.
+
+### Caveats
+
+- Network latency may impact performance.
+- Strapi cannot provide security for external databases.
+- Strapi cannot provide support for external databases.
+
+:::
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.1+`.
+- Credentials for external database.
+- If using an existing database,
+- the schema must match the Strapi project schema.
+
+:::
+
+## Configuration
+
+It is important to ensure that the project `./config/database.js` or `./config/database.ts` file matches the configuration found in the [Environment variables in database configurations](https://docs.strapi.io/dev-docs/configurations/database#environment-variables-in-database-configurations) section.
+
+:::tip
+Before pushing changes, it is recommended to log into the Strapi Cloud project dashboard to add the environment variables.
+
+:::
+
+In the Strapi Cloud project dashboard, navigate to the `Variables` section under the `Settings` tab and click `Add variable`. Once the variables are added, click `Save`.
+
+![environment variables](https://res.cloudinary.com/dz7knyfbp/image/upload/v1681936336/env_var_rw55uy.png)
+
+The following environment variables will need to be set:
+
+| Variable                           | Value            |
+| ---------------------------------- | ---------------- |
+| `DATABASE_CLIENT`                  | your_db          |
+| `DATABASE_HOST`                    | your_db_host     |
+| `DATABASE_PORT`                    | your_db_port     |
+| `DATABASE_NAME`                    | your_db_name     |
+| `DATABASE_USERNAME`                | your_db_username |
+| `DATABASE_PASSWORD`                | your_db_password |
+| `DATABASE_SSL`                     | true             |
+| `DATABASE_SSL_REJECT_UNAUTHORIZED` | false            |
+| `DATABASE_SCHEMA`                  | public           |
+
+:::tip
```

Comment:
```suggestion
:::caution
```
I think it should be a caution much more than a tip because not following it could lead to unexpected consequences. Tips are more for alternate ways of doing things and can be skipped, while the reader should not skip caution callouts 🙂